CMakeLists.txt

```diff
@@ -6,12 +6,14 @@
 # -----------------------------------------------------------------
 
 option(HUNTER_ENABLED "Enable Hunter package manager support" OFF)
-include("cmake/HunterGate.cmake")
-HunterGate(
-  URL "https://github.com/ruslo/hunter/archive/v0.23.34.tar.gz"
-  SHA1 "70287b1ffa810ee4e952052a9adff9b4856d0d54"
-)
-
+option(USE_MKL "Use Intel MKL for blas/lapack" OFF)
+if (HUNTER_ENABLED)

```

Comment:
don't remove the HUNTER_ENABLED option, HunterGate.cmake itself handles the HUNTER_ENABLED option. So no need for the new extra `if()` block

Add just your new option (change of one line)